Client/Frontend/Home/HistoryPanel.swift

```diff
@@ -90,18 +90,20 @@ class HistoryPanel: SiteTableViewController, HomePanel {
         }
     }
 
-    @objc func refresh() {
-        self.refreshControl?.beginRefreshing()
+    func refreshData(completion: (()->())? = nil) {
```

Comment:
So we have `refresh`, `refreshData`, `reloadData`, and `refetchData`? That's rather confusing -- could we either rename them to something more descriptive, or add doc comments explaining what each does?